cmd/editorconfig/main.go

```diff
@@ -13,10 +13,8 @@ import (
 	"github.com/editorconfig/editorconfig-core-go/v2"
 )
 
-const (
-	// Version indicates the current version number
-	Version = "2.1.1"
-)
+// Version indicates the current version number
+const Version = "2.1.1"
```

Comment:
Shouldn't this constant be converted into a variable and be populated automatically by Goreleaser?

Reply:
I've tried that... but `core-test` wants a real value. It could probably be faked in the Makefile.

Reply:
No problem then 👍

Reply:
I've put a fake version in the Makefile.